podcasts/DownloadManager.swift

```diff
@@ -231,6 +231,45 @@ class DownloadManager: NSObject, FilePathProtocol {
         }
     }
 
+    func downloadParallelToStream(of episode: BaseEpisode) -> AVPlayerItem? {
+        guard let playbackItem = PlaybackItem(episode: episode).createPlayerItem() else {
+            return nil
+        }
+
+        guard FeatureFlag.cachePlayingEpisode.enabled,
+              !episode.videoPodcast(),
+              let urlAsset = playbackItem.asset as? AVURLAsset,
+              !urlAsset.url.isFileURL // only  start download if it's a remote file that we are playing
+        else {
+            return playbackItem
+        }
+        #if !os(watchOS)
+        Task {
+            //TODO: check for the following: download is not happening already because of auto download, cancel the auto download and use this file
```

Comment:
Could you please create an issue so we can keep track of these tasks?

I believe it's better than having a `TODO` here. :)

Reply:
You are correct this TODO comment  is no longer needed.

Reply:
Will do.

Reply:
Ticket here: https://github.com/Automattic/pocket-casts-ios/issues/1792

Reply:
I was thinking of a more generic Issue, such as "Downloading streaming episodes with AVMutableComposition". Then we can add subtasks there, like this one.

Reply:
Like this: https://github.com/Automattic/pocket-casts-ios/issues/1795 ?